core/src/repair_service.rs

```diff
@@ -518,20 +532,38 @@ impl RepairService {
         slot: Slot,
         slot_meta: &SlotMeta,
         max_repairs: usize,
+        now_timestamp: u64,
+        defer_threshold_ticks: u64,
```

Comment:
Having two `u64` args next to each other isn't very type-safe.
Is this function always invoked with the same `const`?
Is there any advantage to passing the `const` as an argument instead of just using the `const` value inside the function?

Reply:
The advantage is that blockstore doesn't have a `const` value representing a network delay explicitly baked into the code.

Reply:
I am suggesting to remove `defer_threshold_ticks` arg from `generate_repairs_for_slot` defined here and instead inside the function body just use the `const` which is already defined in this same file.

This wouldn't add any `const` to blockstore.

Reply:
was used in a test, but was unnecessary. changed.